documentation/book/oauth/proc-oauth-kafka-config.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth-strimzi-config.adoc
+
+[id='proc-oauth-kafka-config-{context}']
+= Configuring OAuth for Kafka components
+
+This procedure describes how to configure Kafka components to use {oauth} authentication using an authorization server.
+
+You can configure authentication for:
+
+* Kafka Connect
+* Kafka Mirror Maker
+* Kafka Bridge
+
+In this scenario, the Kafka component and the authorization server are running in the same cluster.
+
+.Prerequisites
+
+* AMQ Streams and Kafka is running
+* {oauth} authorization server is deployed and configured for OAuth access to Kafka brokers
+* Kafka brokers are configured for {oauth}
+
+.Procedure
+
+. Create an OpenShift client Secret and mount it to the component as an environment variable.
++
+For example, here we are creating a client Secret for the Kafka Bridge:
```

Comment:
Could you please add note that the `clientSecret` key has to contain the client secret in base64 format?